devimint/src/util.rs

```diff
@@ -640,6 +648,14 @@ impl GatewayCli {
                 .collect::<Vec<_>>(),
         ))
     }
+
+    /// Returns the gateway-cli version from clap or default min version
+    pub async fn version_or_default() -> Version {
+        match cmd!(GatewayCli, "--version").out_string().await {
```

Comment:
It looks like `gateway-cli`'s version command is `--version-hash`. Will this just always return `DEFAULT_VERSION`?

Reply:
The gateway CLI supports `gateway-cli version-hash` _and_ `gateway-cli --version` so the method above should work

Reply:
Good question! Clap gives us the version flag with

https://github.com/fedimint/fedimint/blob/4aa64957d9faf0fe204f7d36cd3f44a0d4151284/gateway/cli/src/main.rs#L15